help_to_heat/frontdoor/epc_api.py

```diff
@@ -8,35 +8,25 @@
 
 
 class EPCApi:
-    def __init__(self, token):
-        self.token = token
+    def _basic_auth_header(self):
+        return {"Accept": "application/json", "Authorization": f"Basic {settings.OPEN_EPC_API_TOKEN}"}
 
-    def get_access_token(self):
-        client_id = settings.EPC_API_CLIENT_ID
-        client_secret = settings.EPC_API_CLIENT_SECRET
-        base_url = settings.EPC_API_BASE_URL
-        token_url = f"{base_url}/auth/oauth/token"
-
-        payload = {"grant_type": "client_credentials", "client_id": client_id, "client_secret": client_secret}
-
-        response = requests.post(token_url, data=payload)
-        response.raise_for_status()
-        return response.json().get("access_token")
-
-    def get_address_and_rrn(self, building, postcode):
-        base_url = settings.EPC_API_BASE_URL
-        params = urllib.parse.urlencode({"postcode": postcode, "buildingNameOrNumber": building})
-        url = f"{base_url}/api/assessments/domestic-epcs/search?{params}"
+    def get_address_and_lmk(self, building, postcode):
```

Comment:
What does this actually return? Is it more than just address and LMK?

Reply:
it does return the whole search. can update the method name to reflect this & point to the example json object we have in the project